### DISCUSSION TITLE
cheating

### DISCUSSION THREAD

**Discussion by Rapacioli** [SafeExamBrowser/seb-win-refactoring#251] [Q&A]
I hope this finds you well.
I would like to know if this issue is already corrected: https://www.quora.com/Is-there-a-way-to-cheat-while-using-the-Safe-Exam-Browser-and-the-school-won-t-find-out?ch=15&oid=83148234&share=ab1e5699&srid=us3XPv&target_type=question 
Thank you very much
Best regards

Melina Rapacioli.

**Comment by dbuechel** [comment]
Exams performed with BYOD scenarios are never as secure as exams performed in a professionally managed environment. Regarding the VM check: The new display monitoring feature has the convenient side effect that it also can detect VMs (at least according to our own testing), so if you're not using the latest release version, we highly recommend updating to it.

Aside from that, we're working on several ideas to improve the integrity of SEB in BYOD scenarios, some of them however will be mid- to long-term projects. Using SEB in conjunction with SEB-Server is another way to improve the integrity of your exam setup in the meantime.

**Comment by Rapacioli** [comment]
Dear Damian Büchel:
Thank you very much for your kind answer.
We are using SEB from Moodle.
Best regards
Melina Rapacioli
